portfolio/src/main/webapp/style.css

```diff
@@ -56,13 +66,62 @@ img {
   color: grey;
 }
 
-h1,h3,h4 {
-    text-decoration: underline;
-    font-size: xx-larger;
-    font-family: 'Nanum Pen Script', cursive;
+h1 {
```

Comment:
No need to separate all of the headers into different rules.
You can have them together, just on separate lines like:
```
h1,
h3,
h4 {
  <rules>
}
```

See style guide for more info, https://google.github.io/styleguide/htmlcssguide.html#Selector_and_Declaration_Separation.